app/views/ama_layout/v2/_footer.html.erb

```diff
@@ -22,7 +22,7 @@
       </div>
       <ul class="link-list">
         <li class="link-list__item">
-          <%= link_to "Contact Us", "https://ama.ab.ca/about-ama/contact-us/", class: "link-list__link--white" %>
+          <%= link_to "Contact Us", "https://ama.ab.ca/contact/", class: "link-list__link--white" %>
```

Comment:
Instead of hard coding could we make this one dynamic by using:

```suggestion
          <%= link_to "Contact Us", "#{Rails.configuration.amaabca_site}/contact", class: "link-list__link--white" %>
```